src/app/containers/Include/index.test.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import IncludeContainer from '.';
+import { ToggleContext } from '#contexts/ToggleContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+
+const markup = `<!DOCTYPE html><html lang="en"> <script type="text/javascript">var count=0; function incrementCount(cnt){count=cnt + 1; document.getElementById('count').innerHTML=count;}; function decrementCount(cnt){count=cnt - 1; document.getElementById('count').innerHTML=count;}; </script> <head> <style>html{display: inline-flex;}span{background-color: #333; border-radius: 3px; color: #fff; padding: 1px 5px;}button{background-color: #fff; border: 1px solid currentColor; border-radius: 3px; cursor: pointer; outline: 0;}button:active{background-color: #333; color: #fff;}button, span{margin: 0 2px;}</style> </head> <body> <div> <span id="count">0</span> <button onClick="incrementCount(count)">Increase Count</button> <button onClick="decrementCount(count)">Decrease Count</button> </div></body></html>`;
```

Comment:
What's the idea behind the example markup? :-) Does it have some actual purpose or is it just arbitrary?

Reply:
it's just an arbitrary markup

Reply:
Interestingly/worryingly, the structure of the include markup may be far from arbitrary, I was experiencing issues last week with my IDT1 include work and found that scripts injected via `innerHtml` are disabled for security reasons: https://github.com/facebook/react/issues/8838#issuecomment-314740580 

I'm not sure atm why our IDT2 includes work at the moment 😕 The structure of the markup may be the answer to this question.